src/modules/AppHeader/components/AppHeader.js

```diff
@@ -2,14 +2,23 @@ import React from 'react';
 import styled from 'styled-components';
 import theme from 'styles/theme';
 
-const Wrapper = styled.header`
-  align-items: center;
+const Wrapper = styled.div`
   background: ${theme.color.dark_gray};
   background-image: url(${theme.background.swirl});
   background-size: 4em;
-  display: flex;
-  padding: 1em;
+  display: grid;
+  grid-template-columns: 1fr minmax(auto, 600px) 1fr;
+  grid-template-areas:
+	  ". header-content .";
+  grid-area: header;
+  padding: 1em 0;
   margin: 0 0 2em;
+`;
+
+const Header = styled.header`
+  align-items: center;
+  display: flex;
+  grid-area: header-content;
 
   h1 {
```

Comment:
This would be a good place to move these styles (h1 and img) into separate components if you feel inclined (something like Title and Logo, for example). Elements should not be nested inside styled-components.